base_optional_quick_create/model.py

```diff
@@ -38,16 +39,42 @@ def wrapper(cr, uid, name, context=None):
         return wrapper
 
     def _register_hook(self, cr, ids=None):
```

Comment:
@StefanRijnhart @hbrunn I would not know how to modify ```_register_hook``` keeping its behaviour.
What do you think about describing this incompatibility in README?